string[] new_preds
int32[] pidxs
int32[] oidxs
bool[] label
---
bool success
